themes/bootstrap3/package.json

```diff
@@ -9,6 +9,8 @@
     "chart.js": "^4.4.0",
     "simple-keyboard": "^3.6.0",
     "simple-keyboard-layouts": "^3.0.0",
-    "vanilla-cookieconsent": "3.0.0"
+    "vanilla-cookieconsent": "3.0.0",
+    "datatables.net-dt": "1.13.11",
```

Comment:
Might be nice to keep these in alphabetical order.

Reply:
I  have changed the order.